client/src/pages/Search/MapView/RestaurantPreview.js

```diff
@@ -1,7 +1,84 @@
+import React from "react";
+import ReviewStars from "../../../components/ReviewStars";
+import { IconButton } from "@mui/material";
+import CancelIcon from "@mui/icons-material/Cancel";
+import { Link } from "react-router-dom";
+import YelpLogo from "../../../assets/images/yelp-logo.png";
+import RedirectButton from "../../../components/RedirectButton";
+
+const classes = {
+  previewContainer: {
+    height: "160px",
+    width: "100%",
+    backgroundColor: "rgb(224,224,224)",
+    display: "flex",
+    justifyContent: "space-between",
+  },
+  imageContainer: {
+    height: "100%",
+    width: "100px",
+    marginLeft: "1rem",
```

Comment:
not a fan of the left margin. If we want spacing on the side, then let's also have spacing around the div. Here's what I mean:
![image](https://user-images.githubusercontent.com/51217000/230526240-f6b34a10-444e-405f-9222-99d24fd35d31.png)